grunt_tasks/plugin.js

```diff
@@ -246,25 +211,6 @@ module.exports = function (grunt) {
         });
 
     /**
-     * Create a multi-task for all plugin npm installs.
-     */
-    grunt.registerMultiTask(
-        'plugin-install',
-        'Run npm install in plugin directories',
-        function () {
-            var plugin = this.target,
-                tasks = 'plugin-install.' + plugin + '.tasks';
-
-            this.requiresConfig(tasks);
-
-            // queue the install tasks
-            grunt.config.get(tasks).forEach(function (task) {
-                grunt.task.run(task);
-            });
-        }
-    );
-
-    /**
```

Comment:
Do these removals mean that plugins won't be able to use their own `packages.json` file, anymore?